train.py

```diff
@@ -25,6 +25,7 @@
 )
 
 os.environ["TOKENIZERS_PARALLELISM"] = "false"
+os.environ["WANDB_DISABLED"] = "true"
```

Comment:
for Yolo, we need to activate it. So maybe we could use  a new --wandb-status "disabled" by defaults when it's not yolo?

Reply:
adding a `WANDB_DISABLED`=false in the yolo environment, until we have it integrated properly in the HF jobs.